job_definitions/notify_suppliers_of_brief_withdrawals.yml

```diff
@@ -0,0 +1,54 @@
+{% set environments = ['preview', 'staging', 'production'] %}
+---
+{% for environment in environments %}
+- job:
+    name: "notify-suppliers-of-brief_withdrawals-{{ environment }}"
+    display-name: "Notify suppliers that their responses have been cancelled when a brief has been withdrawn - {{ environment }}"
+    project-type: freestyle
+    description: "Send email notifications to supplier users if they have a completed application to a brief that has been withdrawn."
+    parameters:
+      - string:
+          name: WITHDRAWN_DATE
+          default: null
+          description: "Specify a given date, send all emails for that date"
+      - string:
+          name: BRIEF_ID
+          default: null
+          description: "Specify a single brief, requires withdrawn_date"
+      - bool:
+          name: DRY_RUN
+          default: false
+          description: "List notifications that would be sent without sending the emails"
+    triggers:
+      - timed: "H 8 * * *"
+    publishers:
+      - trigger-parameterized-builds:
+          - project: notify-slack
+            condition: UNSTABLE_OR_WORSE
+            predefined-parameters: |
+              USERNAME=brief-withdrawals
+              JOB=Notify suppliers of brief withdrawals {{ environment }}
+              ICON=:question:
```

Comment:
Do you want to pick a different icon to the 'notify suppliers of new questions and answers' job?

Reply:
Done